docs/source/content/deep_learning/cnn.rst

```diff
@@ -0,0 +1,237 @@
+#############################
+Convolutional Neural Networks
+#############################
+
+.. contents::
+  :local:
+  :depth: 2
+
+
+********
+Overview
+********
+In the last module, we started our dive into deep learning by talking about
+multi-layer perceptrons. In this module, we will learn about **convolutional
```

Comment:
I would define what a perceptron is. I don't think most people would know what that means

Reply:
Thanks! I will link to the other module when available.